README.md

```diff
@@ -1,35 +1,116 @@
-## How it works
+# Connect
 
-The Connectivity component is responsible for the communication with the target system. This component implements the logic of the interaction protocol, receives and sends messages from and to the system, respectively.
+The "Connect" component is responsible for the communication with a target system.
+This component implements the logic of the interaction protocol, receiving and sending messages from and to the system, respectively.
 
-## Queues
+This project includes only an adapter for using the Sailfish service in the th2 packed into a Docker Image.
+This image should be used as a base to implement extensions with the real logic for specific protocols using services in the Sailfish format.
 
-Connectivity has 4 queues for interacting with th2 components. Messages that were received from / sent to the target system will be sent to the following queues:
+As an example, the [th2-conn-generic](https://github.com/th2-net/th2-conn-generic) project implements the extension for connecting via FIX protocol using standard Sailfish's FIX service.
 
-- incoming parsed messages
-- outgoing parsed messages
+## Configuration
+
+This configuration should be specified in the custom configuration block in schema editor.
```

Comment:
This configuration should be specified in the custom configuration block **at the** schema editor.